recipes/cuda-profiler-api/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "cuda-profiler-api" %}
```

Comment:
Shouldn't this be `cuda-profiler-api-dev`? this is just shipping header files which based on our pattern with `nvml` would make this a dev package.

Reply:
Agree with this.

Reply:
We have other cases where `-dev` naming conventions aren't followed. Mostly devtools, but also `cuda-cccl`. I'd rather stay consistent with how this component has shipped previously.

Reply:
But don't we also do `cuda-nvml-dev`?

Reply:
That also matches the given component name and historical naming, which would not be the case here.

Reply:
I'd also like to keep consistency with the existing package names.

Reply:
I am fine with keeping the current names so we have consistency.